apps/mobile/src/components/qrcode/Show.tsx

```diff
@@ -1,11 +1,45 @@
+import { Image, StyleSheet } from 'react-native'
 import { SafeAreaView } from 'react-native-safe-area-context'
 
-import { Text } from '../Themed'
+import { Text, View } from '../Themed'
 
 export function Show() {
     return (
         <SafeAreaView>
-            <Text>This is just a placeholder</Text>
+			<View style={styles.centered}>
+				<View style={styles.message}>
+					<Image
+						source={require("../../assets/images/hidetaka.png")}
+					/>
+					<Text style={styles.title}>Hidetaka Ko</Text>
```

Comment:
Same as my preview comment, in this codebase, we already have the profile data, so please fill the name dynamically from the profile instead of putting the static name